### PR TITLE
EditorNode: enhance open scene error dialog

### PR DESCRIPTION
**Pull request by ghost**
This aims to alleviate the most infuriating dialog of all time (in my opinion):
image

After:
image

(This happens when choosing 'Open Anyway' and still can't open.)
image

### CODE REVIEW

**Comment by Zireael07** [comment]
Can the parser tell us the line where the error happened? (ie. "error while parsing s15.tscn, line 90")

**Comment by Noshyaar** [comment]
I tried but couldn't think of a non-hacky way to get error message that deep. The current code returns only Error enum.

**Comment by Zireael07** [comment]
Well, this PR is still an improvement over the current state of things :)

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -586,6 +586,41 @@ void EditorNode::_dialog_display_file_error(String p_file, Error p_error) {
...
+	if (p_error) {
+
+		current_option = -1;
+		accept->get_ok()->set_text(TTR("I see.."));
+
+		switch (p_error) {
+
+			case ERR_CANT_OPEN: {
+
+				accept->set_text(vformat(TTR("Can't open '%s'"), p_file.get_file()));
```
Comment (akien-mga) [inline editor/editor_node.cpp]: Can you end those messages with a dot?

**Comment by HummusSamurai** [comment]
Definitely the most infuriating dialog of all time. <3

**Pull request merged**